components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/store/JDBCRecoveryDataStore.java

```diff
@@ -105,15 +105,15 @@ public UserRecoveryData loadByCode(String code) throws IdentityRecoveryException
                         namedPreparedStatement.setString(CODE, code);
                     });
         } catch (DataAccessException e) {
-            throw Utils.handleServerException(IdentityRecoveryConstants.ErrorMessages.ERROR_CODE_UNEXPECTED, null, e);
+            throw Utils.handleServerException(IdentityRecoveryConstants.ErrorCodes.UNEXPECTED, null, e);
```

Comment:
its much better to have an overloaded method 
Utils.handleServerException(IdentityRecoveryConstants.ErrorCodes.UNEXPECTED, e);
instead of
Utils.handleServerException(IdentityRecoveryConstants.ErrorCodes.UNEXPECTED, null, e);